### PR TITLE
Added loader function for coefficients

### PR DESCRIPTION
**Pull request by PhilippKopp**

### CODE REVIEW

File: include/eos/morphablemodel/coefficients.hpp [hunk 1]
```diff
@@ -50,6 +50,26 @@ inline void save_coefficients(std::vector<float> coefficients, std::string filen
...
+ *
+ * @param[in] filename The file to write.
+ * @throws std::runtime_error if unable to open the given file for reading.
+ * @return Returns vector of floats.
+ */
+inline std::vector<float> load_coefficients(std::string filename)
+{
+    std::vector<float> coefficients;
+    std::ifstream file(filename);
+    if (file.fail())
```
Comment (patrikhuber) [inline include/eos/morphablemodel/coefficients.hpp]: I see you copied this from the `save_` method. I can't believe there was still a check in eos that used `.fail()`! I changed all of them but must've missed this one. Please change this to just `if (file)`, and if you could change the `save_` method too!

File: include/eos/morphablemodel/coefficients.hpp [hunk 2]
```diff
@@ -50,6 +50,26 @@ inline void save_coefficients(std::vector<float> coefficients, std::string filen
     output_archive(cereal::make_nvp("shape_coefficients", coefficients));
 };
 
+/**
+ * Loads coefficients (for example PCA shape coefficients) from a json file.
+ *
+ * @param[in] filename The file to write.
+ * @throws std::runtime_error if unable to open the given file for reading.
```
Comment (patrikhuber) [inline include/eos/morphablemodel/coefficients.hpp]: Put the `@throws` after the `@return` please

**Comment by patrikhuber** [comment]
See comments, LGTM otherwise 👍 !

**Comment by PhilippKopp** [comment]
Thanks for the comments! Committed correction.

**Pull request merged**